frontend/lodh-fall/lodh.css

```
@@ -0,0 +1,2226 @@
+.season-tabs {
+  display: flex;
+  justify-content: center;
+  gap: 10px;
+
+  margin-bottom: 30px;
+}
+
+.season-tab {
+  padding: 13px 24px;
+
+  border: 1px solid rgba(23, 56, 43, 0.15);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)